src/agentscope/service/reasoning/wolfram_alpha.py

```diff
@@ -0,0 +1,255 @@
+# -*- coding: utf-8 -*-
+"""Query Wolfram Alpha API"""
+from typing import Any, Dict  # Corrected import order
+import requests  # Corrected import order
+from loguru import logger  # Corrected import order
+from agentscope.service.service_response import (
+    ServiceResponse,
+    ServiceExecStatus,
+)
+
+
+def query_wolfram_alpha_short_answers(
+    api_key: str,
+    query: str,
+) -> ServiceResponse:
+    """
+    Query the Wolfram Alpha Short Answers API.
+
+    Args:
+        api_key (`str`):
+            Your Wolfram Alpha API key.
+        query (`str`):
+            The query string to search.
+
+    Returns:
+        `ServiceResponse`: A dictionary with two variables: `status` and
+        `content`. The `status` variable is from the ServiceExecStatus enum,
+        and `content` is a dictionary containing
+        the result or error information,
```

Comment:
What are the information and fields in the status?

Reply:
example outputs are now provided in the docstring